package.json

```diff
@@ -14,6 +14,7 @@
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test:e2e": "npx cypress run",
     "test:e2e:interactive": "npx cypress open",
+    "test:e2e:ci": "npm run build && run-p --race start test:e2e",
     "test:storybook": "npx cypress run --project ./.storybook/",
```

Comment:
I know this is WIP. Please remember we now have 2 sets of cypress tests to run, the second is `npm run test:storybook`. This wasn't true when the issue was written hence this prompt